src/main/resources/db/changelog/v.1.0/db.changelog_v.1.0.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="1" author="dmt">
```

Comment:
pls use long format for changeSet id .
just example:

`28.05.2020-dmt-01` for first changeSet`

`28.05.2020-dmt-02` for second changeSet`

and etc 

pls check other places